hugr-model/src/v0/text/print.rs

```diff
@@ -612,6 +612,15 @@ impl<'p, 'a: 'p> PrintContext<'p, 'a> {
                 self.print_text("meta");
                 Ok(())
             }
+            Term::Float { value } => {
+                // The debug representation of a float always includes a decimal point.
+                self.print_text(format!("{:?}", value.into_inner()));
+                Ok(())
+            }
+            Term::FloatType => {
```

Comment:
looks like float type isn't being covered, pls add to test

Reply:
Should be covered now; can't check because the codecov CI links seem to be broken...